<launch>
    <node pkg="tf" type="static_transform_publisher" name="eelink2camera" args="-0.0051 0.07277 -0.1265 0.705564 0.0008812 0.7075963 0.038551 ee_link camera  100"/>
    <node name="aruco_pub" pkg="shallow_depth_insertion" type="ArucoPose_publisher" required="true" output="screen"/>
    <node name="aruco_tf" pkg="shallow_depth_insertion" type="ArucoTF_broadcaster" required="true" output="log" />
</launch>
